### PR TITLE
Try to enable Gradle Daemon again

### PR DESCRIPTION
**Pull request by Goooler**
## Description

Gradle Daemon was disabled for PR Build jobs in #1584 as some there were flaky tests on Windows, re-enable it for now.

## Checklist

- [x] PR description added
- [ ] tests are added
- [ ] KtLint has been applied on source code itself and violations are fixed
- [ ] [documentation](https://pinterest.github.io/ktlint/) is updated
- [ ] `CHANGELOG.md` is updated

In case of adding a new rule:
- [ ] Rule is added to [rules documentation](https://pinterest.github.io/ktlint/rules/standard/)

### CODE REVIEW

**Comment by Goooler** [comment]
Merge this after #1884.

File: build.gradle.kts [hunk 1]
```diff
@@ -21,13 +20,6 @@ allprojects {
             logger.warn("Skipping tests for task '$name' as system property 'skipTests=$skipTests'")
         }
 
-        retry {
-            if (System.getenv().containsKey("CI")) {
-                maxRetries.set(3)
-                maxFailures.set(20)
-            }
-        }
```
Comment (Goooler) [inline build.gradle.kts]: We can enable this `retry` if flack tests occur again, just remove it for now.

**Comment by paul-dingemans** [comment]
I am really to go enjoy the improved speed of the build. Tnx!

**Comment by Goooler** [comment]
My pleasure.

**Pull request merged**